content/doc/book/pipeline/development.adoc

```diff
@@ -0,0 +1,145 @@
+---
+layout: section
+title: Pipeline Development Tools
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Pipeline Development Tools
+
+In this section, we'll discuss tools and resources
+that help speed development of Jenkins Pipelines.
+
+== Blue Ocean Editor (Declarative only)
+
+The
+<<../blueocean/pipeline-editor#, Blue Ocean Pipeline Editor>> provides a WYSIWYG
+way to create Declarative Pipelines. It gives you a structural view of all the stages,
+parallel branches, and steps in your Pipeline. It validates Pipeline changes as they are
```

Comment:
I cannot cite specific rules/guidelnies, but I personally try to avoid sentences where the subject is a pronoun, requiring surrounding context to define the pronoun.

My reasoning is that I think it's confusing, but also indicates statements which could be made more clear or less repetitive.